src/i18n/locales/en-US.json

```diff
@@ -271,6 +271,34 @@
   "cmps.designer.taro.ConnectTab.filePaths": "File Paths",
   "cmps.designer.taro.ConnectTab.hexStrings": "HEX",
   "cmps.designer.taro.ConnectTab.base64Strings": "Base64",
+  "cmps.designer.taro.ActionsTab.MintAsset.title": "Minting",
```

Comment:
This key is, as well as others, are not used by any component. Please check to make sure there are only strings that are in use in this file.

Reply:
Done